tests/policy/data/main.fmf

```diff
@@ -0,0 +1,27 @@
+test: /bin/true
+
+/basic:
+
+/full:
+  contact:
+    - foo
+    - bar
+
+  environment:
+    FOO: BAR
+    QUX: QUUX
+
+  check:
+    - how: avc
+      result: info
+    - how: dmesg
+      failure-pattern: ".*"
+
+/value-source:
+  /no-custom-duration:
+
+  /custom-duration:
+    duration: 5m +5m
+
+  /default-duration:
```

Comment:
The naming is super confusing. Took me some time to match this with the policy rules. What about using `/default-duration` for the first case when we use the actual default and perhaps `/same-as-default` for the user defined one?

Reply:
Addressed in e8e84463